ethToElrond/v2/ethToElrond/steps/semiIntegrated_test.go

```diff
@@ -50,6 +54,10 @@ func getWasProposedTransferSignedValue(mode testMode) bool {
 	return mode&proposedTransferSigned > 0
 }
 
+func getReachMaxRetriesAllowed(mode testMode) bool {
```

Comment:
I don`t think we need a disabling trigger for the retry mechanism in tests if we don`t have it implemented also in the code. What do you think ?

Reply:
agree with you. I'll remove it. The only thing needed then is to reset the counter on each test.